packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -65,14 +65,18 @@ class PointerEventConverter {
     );
   }
 
-  /// Expand the given packet of pointer data into a sequence of framework
+  static String _formatErrorMessage(ui.PointerChange change, String msg) => 'Invalid event \'$change\': $msg';
+
+
+  /// Format the given packet of pointer data into a framework
   /// pointer events.
   ///
   /// The `devicePixelRatio` argument (usually given the value from
   /// [dart:ui.Window.devicePixelRatio]) is used to convert the incoming data
   /// from physical coordinates to logical pixels. See the discussion at
   /// [PointerEvent] for more details on the [PointerEvent] coordinate space.
-  static Iterable<PointerEvent> expand(Iterable<ui.PointerData> data, double devicePixelRatio) sync* {
+  static List<PointerEvent> format(List<ui.PointerData> data, double devicePixelRatio) {
```

Comment:
renaming `expand` to `format` is also a breaking change. not sure that one is worth it.

Reply:
Is there a use case where developer will come in and use this function directly?

Reply:
fixed